test/OpenTelemetry.Instrumentation.SqlClient.Tests/SqlClientIntegrationTests.cs

```diff
@@ -104,6 +105,126 @@ public void SuccessfulCommandTest(
         }
     }
 
+    private static void VerifyActivityData(
```

Comment:
I moved some methods to this file because the integration test was the only remaining thing using them.

I still need to decide what to do with this integration test. The new test suite could fairly easily be turned into proper integration tests replacing this one.